### PR TITLE
Step 2 of author formatting changes: run v2 of prepro

### PR DESCRIPTION
**Pull request by anna-parker**
Test https://github.com/loculus-project/loculus/pull/2986

### CODE REVIEW

File: loculus_values/values.yaml [hunk 1]
```diff
@@ -1101,6 +1102,13 @@ organisms:
           <<: *preprocessingConfigFile
           nextclade_dataset_server: https://raw.githubusercontent.com/nextstrain/nextclade_data/ebola/data_output
           nextclade_dataset_name: nextstrain/ebola/zaire
+      - <<: *preprocessing
+        version: 2
+        dockerTag: commit-6e05226
```
Comment (corneliusroemer) [inline loculus_values/values.yaml:1107]:
```suggestion
```

File: pathoplexus_app/values.yaml [hunk 2]
```diff
@@ -1,3 +1,3 @@
-loculusVersion: 712fde0a6c1f9b8386965c470a98add9672b2ea2
+loculusVersion: 6e0522666f7fef744fb348513926c114a18a7ee9
```
Comment (corneliusroemer) [inline pathoplexus_app/values.yaml:1]:
```suggestion
loculusVersion: 07f7872ce2808c45e778fa5c65f5d6989fdb7c12
```

File: loculus_values/values.yaml [hunk 3]
```diff
@@ -1096,6 +1096,13 @@ organisms:
   ebola-zaire:
     <<: *defaultOrganismConfig
     preprocessing:
+      - <<: *preprocessing
+        version: 1
+        dockerTag: commit-6e05226
```
Comment (anna-parker) [inline loculus_values/values.yaml:1101]:
```suggestion
        dockerTag: commit-9704260
```

File: loculus_values/values.yaml [hunk 4]
```diff
@@ -435,6 +435,10 @@ defaultOrganismConfig: &defaultOrganismConfig
         header: Authors
         truncateColumnDisplayTo: 15
         ingest: ncbiSubmitterNames
+        preprocessing:
```
Comment (anna-parker) [inline loculus_values/values.yaml:438]: sadly this means that while we are waiting for the sequences to be reprocessed the old pipeline will not be usable (throw an error due to unknown function) unless we temporarily add this to the preprocessing pipeline v1 as a no-op.

Comment (corneliusroemer) [reply loculus_values/values.yaml:438]: aah, good catch! dying think it's a problem since the probability of submission is low no?

Comment (anna-parker) [reply loculus_values/values.yaml:438]: yeah probably I created this just now as a workaround: https://github.com/loculus-project/loculus/pull/3065 but potentially we actually do want submissions to not work in the off-chance someone submits with incorrect authors format while we are doing this

**Pull request closed without merging**